qemu/qemu.go

```diff
@@ -1020,6 +1020,13 @@ func (config *Config) appendKnobs() {
 		} else {
 			config.qemuParams = append(config.qemuParams, "mlock=off")
 		}
+	} else {
+		// QEMU always locks memory anyway if realtime is not defined
+		if config.Knobs.Mlock == false {
+			//Enable realtime anyway just to get the right swapping behaviour
```

Comment:
its a nit, but I would word this comment differently. I would say "in order to turn mlock off we need the -realtime option as well"